test/e2e/webdriver/driver.js

```diff
@@ -38,12 +38,35 @@ class Driver {
     return element
   }
 
-  findElements (locator) {
-    return this.driver.wait(until.elementsLocated(locator), this.timeout)
+  async findClickableElement (locator) {
+    const element = await this.findElement(locator)
+    await Promise.all([
+      this.driver.wait(until.elementIsVisible(element), this.timeout),
+      this.driver.wait(until.elementIsEnabled(element), this.timeout),
```

Comment:
Why do we need this twice?

Reply:
The first is `elementIsVisible`, the second is `elementIsEnabled`

Reply:
Ah, I missed that bit